### PR TITLE
Feature/clan6 210 re design widgets so they are more compact

### PR DESCRIPTION
**Pull request by hzietek**

### CODE REVIEW

File: cogboard-webapp/src/components/Widget/helpers.js [hunk 1]
```diff
@@ -12,22 +19,49 @@ export const renderCardContent = (
...
   disabled,
   id,
-  type
+  type,
+  status,
+  expandContent,
+  expanded,
+  handleExpandClick
 ) => {
+  console.log(content);
```
Comment (thisisui) [inline cogboard-webapp/src/components/Widget/helpers.js]: to be removed

File: cogboard-webapp/src/components/Widget/styled.js [hunk 2]
```diff
@@ -81,4 +83,64 @@ export const StyledCardContent = styled(CardContent)`
...
+  transition: transform 100ms linear;
+`;
+
+export const StyledStatusIconButton = styled.a`
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  transition: 0.2s background-color;
+  border-radius: 2px;
+  padding: 5px 0 5px 0;
```
Comment (thisisui) [inline cogboard-webapp/src/components/Widget/styled.js]: 5px 0 will do the work

File: cogboard-webapp/src/components/Widget/helpers.js (hunk 1 above)
Comment (hzietek) [reply cogboard-webapp/src/components/Widget/helpers.js]: removed

File: cogboard-webapp/src/components/Widget/styled.js (hunk 2 above)
Comment (hzietek) [reply cogboard-webapp/src/components/Widget/styled.js]: done

File: cogboard-webapp/.env [hunk 3]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_PORT=8092
+REACT_APP_WS_PORT=9001
```
Comment (tMaxx) [inline cogboard-webapp/.env]: I don't think you should've committed this file here.

File: cogboard-webapp/src/components/StatusIcon.js [hunk 4]
```diff
@@ -1,24 +1,34 @@
...
+  const sizeProp = status === 'IN_PROGRESS' ? { size: 26 } : { fontSize: size };
 
   return IconComponent && <IconComponent color="inherit" {...sizeProp} />;
 };
 
 export default StatusIcon;
 
 StatusIcon.propTypes = {
-  status: string.isRequired
+  status: string
```
Comment (tMaxx) [inline cogboard-webapp/src/components/StatusIcon.js]: Similar entry for `size`?

File: cogboard-webapp/src/components/widgets/types/TextWidget/index.js [hunk 5]
```diff
@@ -36,7 +36,6 @@ const TruncatedText = ({ isVertical, parentDimensions, children }) => {
 const TextWidget = ({ text, textSize, isVertical }) => {
   const targetRef = useRef();
   const centerWrapperDimensions = useSize(targetRef);
-
```
Comment (tMaxx) [inline cogboard-webapp/src/components/widgets/types/TextWidget/index.js]: Don't remove vertical whitespace, they want to be present and loved ;)

File: cogboard-webapp/.env (hunk 3 above)
Comment (hzietek) [reply cogboard-webapp/.env]: removed

File: cogboard-webapp/src/components/StatusIcon.js (hunk 4 above)
Comment (hzietek) [reply cogboard-webapp/src/components/StatusIcon.js]: added

File: cogboard-webapp/src/components/StatusIcon.js [hunk 6]
```diff
@@ -1,24 +1,35 @@
 import React from 'react';
 import { string } from 'prop-types';
 
-import { Warning } from '@material-ui/icons';
+import { Warning, Check } from '@material-ui/icons';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
+import ShowChartIcon from '@material-ui/icons/ShowChart';
+import BlockIcon from '@material-ui/icons/Block';
+import PortableWifiOffIcon from '@material-ui/icons/PortableWifiOff';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
```
[the remark this answers is not in the document]
Comment (hzietek) [reply cogboard-webapp/src/components/StatusIcon.js]: Done

File: cogboard-webapp/src/components/StatusIcon.js [hunk 7]
```diff
@@ -1,24 +1,35 @@
...
 
   return IconComponent && <IconComponent color="inherit" {...sizeProp} />;
 };
 
 export default StatusIcon;
 
 StatusIcon.propTypes = {
-  status: string.isRequired
+  status: string,
+  size: string
```
[the remark this answers is not in the document]
Comment (hzietek) [reply cogboard-webapp/src/components/StatusIcon.js]: Added

File: cogboard-webapp/src/components/Widget/styled.js [hunk 8]
```diff
@@ -68,8 +70,8 @@ StyledCard.propTypes = {
   columns: number.isRequired,
   goNewLine: bool.isRequired,
   rows: number.isRequired,
-  status: string.isRequired,
-  theme: object.isRequired
+  theme: object.isRequired,
+  status: string
```
[the remark this answers is not in the document]
Comment (hzietek) [reply cogboard-webapp/src/components/Widget/styled.js]: Added

File: cogboard-webapp/src/components/StatusIcon.js [hunk 9]
```diff
@@ -1,24 +1,35 @@
...
-  IN_PROGRESS: Loader
+  IN_PROGRESS: Loader,
+  OK: Check,
+  UNKNOWN: HelpOutlineIcon,
+  UNSTABLE: ShowChartIcon,
+  FAIL: BlockIcon
 };
 
-const StatusIcon = ({ status }) => {
+const StatusIcon = ({ status, size }) => {
```
[the remark this answers is not in the document]
Comment (hzietek) [reply cogboard-webapp/src/components/StatusIcon.js]: Sizes are mapped now

File: cogboard-webapp/src/components/Widget/index.js [hunk 10]
```diff
@@ -110,6 +114,10 @@ const Widget = ({ id, index }) => {
     closeConfirmationDialog();
   };
 
+  const handleExpandClick = () => {
```
[the remark this answers is not in the document]
Comment (hzietek) [reply cogboard-webapp/src/components/Widget/index.js]: Done

**Pull request merged**